docs/delivery/README.md

```diff
@@ -0,0 +1,121 @@
+# Delivery Design
+
+This document synthetizes the [error handling design document](https://docs.google.com/document/d/1qRrzGoHJQO-oc5p-yRK8IRfugd-FM_PXyM7lN5kcqks).
+
+## Problem
+
+Sending events can fail for a variety of reasons (downstream system is down, application logic rejects the invalid message, a runtime exception occurs, etc...) but right now there is no way to control or define the expected behavior in these situations.
+
+## Requirements
+
+* Be able to handle events that failed to be delivered
+    * to channel subscribers.
+    * to source sink.
+    * to broker/triggers.
+* Be able to identify a message couldn’t be delivered (Observability)
+* Be able to leverage existing native error handling mechanisms (eg. dead letter queues).
+* Be able to redirect of error'ed events from a channel.
+
+### Out of Scope
+
+* Security: No security model has been defined yet at the Knative eventing level. Native brokers might define their own security model and Knative eventing should not prevent using it.
+
+## Day 1 Proposal
+
+### Error Sink
+
+Channels are responsible for forwarding received messages to subscribers. When they fail to do so, they are responsible for sending the failing messages to an **error sink**. It could be a channel but it does not have to.
+
+Similarly Event Sources are responsible sending events to a sink and when they fail to do so, they are responsible for sending the failing events to an **error sink**.
+
+### Dead-Letter Channel
+
+Channel implementations might leverage the existing native error handling support they provide, usually a dead letter channel, to forward failed messages to the error sink. In that case, the error sink might be realized by creating a subscription on the error channel.
```

Comment:
_Channel implementations might leverage the existing native error handling support they provide, usually a dead letter channel_ 

Could we add a link to the EIP definition?

and perhaps add something like

```
Knative Channel implementations may leverage existing platform native error handling support they might provide, like a a [_Dead Letter Channel_](https://www.enterpriseintegrationpatterns.com/patterns/messaging/DeadLetterChannel.html), to forward failed messages from their _Dead Letter Channel_ to the configured error sink. 
```